src/maliput_malidrive/builder/direction_usage_builder.h

```diff
@@ -77,7 +77,7 @@ class DirectionUsageBuilder {
   ///
   /// @param state is the maliput::api::rules::DirectionUsageRule::State::Type to be parsed.
   /// @return An optional of maliput::api::rules::DirectionUsageRule::State::Type with the parsed type.
-  /// @throws maliput::common::assertion_error When `state` is unrecognized.
+  /// @throws maliput::common::state_provider_error When `state` is unrecognized.
```

Comment:
this should be rulebook_error.

State provider error should be reserved to phase and rule state providers.

Reply:
Done.